tests/external_population_test.go

```diff
@@ -44,22 +45,31 @@ var _ = Describe("Population tests", func() {
 
 	sampleGVR := schema.GroupVersionResource{Group: populatorGroupName, Version: populatorAPIVersion, Resource: populatorResource}
 	apiGroup := populatorGroupName
+	dummyAPIGroup := "dummy.populator.io"
 	dataSourceRef := &corev1.TypedLocalObjectReference{
 		APIGroup: &apiGroup,
 		Kind:     populatorKind,
 		Name:     samplePopulatorName,
 	}
+	dummySourceRef := &corev1.TypedLocalObjectReference{
+		APIGroup: &dummyAPIGroup,
+		Kind:     "Dummy",
+		Name:     "dummyname",
+	}
```

Comment:
isn't it an error for source and sourceRef to not match?

Reply:
they'll match here, just switching to a dummy one so we don't trigger the testing populator